circle/circle.go

```diff
@@ -15,6 +16,10 @@ Perimeter: 50.27, square: 201.06
 */
 func main() {
 	fmt.Println("The program prints the perimeter and the square of a circle given the radius.")
+	fmt.Print("Enter the radius: ")
+	var r float64
+	fmt.Scan(&r)
+	fmt.Printf("Perimeter: %.2f, square: %.2f", 2*r*math.Pi, math.Pi*r*r)
```

Comment:
(same as in `speed.go`) It's better to assign the expressions to the variables first, with the names reflecting the purpose of the expressions.